pkg/repository/git.go

```diff
@@ -256,3 +258,22 @@ func (ogr oktetoGitWorktree) Status(ctx context.Context, localGit LocalGitInterf
 
 	return oktetoGitStatus{status: status}, nil
 }
+
+func FindTopLevelGitDir(workingDir string, fs afero.Fs) (string, error) {
```

Comment:
I've used this approach because the `git-go` library does not support `git rev-parse --show-toplevel` (See https://github.com/go-git/go-git/issues/74) and we would have a hard dependency on the `git` binary. This approach is also used in this [google project](https://github.com/GoogleContainerTools/skaffold/pull/275/files#diff-566b8bbd20b2d5f058993d060ab8bbd46a938b6186ac5bbcfd11adf271f807eeR96-R113)